src/main/resources/db/migration/0400/V402__Activities.sql

```diff
@@ -0,0 +1,50 @@
+-- Drop unused file metadata fields that were added as part of the document producer port.
+ALTER TABLE files DROP COLUMN file_date;
+ALTER TABLE files DROP COLUMN height;
+ALTER TABLE files DROP COLUMN width;
+
+CREATE TABLE accelerator.activity_types (
+    id INTEGER PRIMARY KEY,
+    name TEXT NOT NULL UNIQUE
+);
+
+CREATE TABLE accelerator.activities (
+    id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    project_id BIGINT NOT NULL REFERENCES projects ON DELETE CASCADE,
+    activity_type_id INTEGER NOT NULL REFERENCES accelerator.activity_types,
+    activity_date DATE NOT NULL,
+    description TEXT NOT NULL,
+    is_highlight BOOLEAN NOT NULL,
+    created_by BIGINT NOT NULL REFERENCES users,
+    created_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    modified_by BIGINT NOT NULL REFERENCES users,
+    modified_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    verified_by BIGINT REFERENCES users,
+    verified_time TIMESTAMP WITH TIME ZONE
+);
+
+CREATE INDEX ON accelerator.activities (project_id);
+
+CREATE TABLE accelerator.activity_media_types (
+    id INTEGER PRIMARY KEY,
+    name TEXT NOT NULL UNIQUE
+);
+
+CREATE TABLE accelerator.activity_media_files (
+    file_id BIGINT NOT NULL REFERENCES files ON DELETE CASCADE,
+    -- No ON DELETE CASCADE on activity ID because we need to delete the files from storage
+    activity_id BIGINT NOT NULL REFERENCES accelerator.activities,
+    activity_media_type_id INTEGER NOT NULL REFERENCES accelerator.activity_media_types,
+    is_cover_photo BOOLEAN NOT NULL,
+    captured_date DATE NOT NULL,
+    caption TEXT,
+    geolocation GEOMETRY(POINT),
+
+    PRIMARY KEY (file_id)
+);
+
+-- Only allow one cover photo per activity.
+CREATE UNIQUE INDEX one_cover_per_activity ON accelerator.activity_media_files (activity_id)
+WHERE is_cover_photo;
```

Comment:
Should there also be a check to ensure the cover photo is a photo and not a video?